processing/src/main/java/org/apache/druid/query/monomorphicprocessing/SpecializationService.java

```diff
@@ -209,14 +349,20 @@ T specialize(ImmutableMap<Class<?>, Class<?>> classRemapping)
     @SuppressWarnings("unchecked")
     private Class<T> defineClass(String specializedClassName, byte[] specializedClassBytecode)
     {
-      return (Class<T>) UNSAFE.defineClass(
-          specializedClassName,
-          specializedClassBytecode,
-          0,
-          specializedClassBytecode.length,
-          prototypeClass.getClassLoader(),
-          prototypeClass.getProtectionDomain()
-      );
+      if (DEFINE_CLASS_NOT_SUPPORTED_EXCEPTION != null) {
+        throw new UnsupportedOperationException(DEFINE_CLASS_NOT_SUPPORTED_EXCEPTION);
+      }
+
+      try {
+        return (Class<T>) DEFINE_CLASS.invokeExact(
+            prototypeClass,
+            specializedClassBytecode,
+            specializedClassName
+            );
+      }
+      catch (Throwable t) {
+        throw new UnsupportedOperationException("Unable to define specialized class: " + specializedClassName, t);
```

Comment:
`t` should be on a separate line

Reply:
checkstyle did not complain, did I miss anything?

Reply:
There is no appropriate rule in Checkstyle for this element of Druid style

Reply:
I'm not sure what rule I broke then, since this fits into the line length

Reply:
Right, sorry. I thought it's on two separate lines.